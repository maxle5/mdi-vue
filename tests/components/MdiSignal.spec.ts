
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiSignal from "../../src/components/MdiSignal.vue";

test("MdiSignal snapshot", () => {
  const wrapper = mount(MdiSignal, {});
  expect(wrapper.html()).toMatchSnapshot();
});
