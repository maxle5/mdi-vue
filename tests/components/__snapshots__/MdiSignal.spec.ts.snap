// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiSignal snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M3,21H6V18H3M8,21H11V14H8M13,21H16V9H13M18,21H21V3H18V21Z" style="fill: currentColor;"></path>
</svg>"
`;
